releasenotes/notes/time-evo-framework-9d58827bdbbebd62.yaml

```
@@ -2,13 +2,12 @@
 features:
   - |
     Interfaces for the unified framework for Quantum Time Evolution are introduced.
-    :class:`~qiskit.algorithms.evolvers.EvolutionProblem` defines an evolution problem and
+    :class:`~qiskit.algorithms.EvolutionProblem` defines an evolution problem and
     can be passed to any evolution algorithm available in the framework.
-    :class:`~qiskit.algorithms.evolvers.Evolver` interface serves as a base for
-    any time evolution algorithm for evolving quantum states, including evolutions based on
+    :class:`~qiskit.algorithms.ImaginaryEvolver` and
+    :class:`~qiskit.algorithms.RealEvolver` are derived interfaces for
```

```suggestion
    :class:`~qiskit.algorithms.RealEvolver` are interfaces for
```
I think we can leave off derived since we are not really exposing the base class directly and its a level of detail that I don't think users would really need to know right. They would work with a RealEvolver or an ImaginaryEvolver instance.